### ISSUE TITLE
Recherche: The label for one of the filters is in English on the French page

### ISSUE THREAD

**Issue by sarasuri** [mmiron-HC/WST-RSAMS#162]
**Page:** Recherche
**URL:** https://rsams.dev.cloud.openplus.ca/fr/recherche/site and https://recalls-rappels.canada.ca/fr/recherche/site
**User Agent:** Firefox (v99)
**Language:** French

**Issue Description:** Under "Classe de rappel", label for one of the filters is in English on the French page.

![image](https://user-images.githubusercontent.com/12256067/162264003-7cf1b943-290f-4a74-8613-d84229316187.png)

**Steps to reproduce:**

1. Go to the above URL
2. Visual scanning

**Expected results:** The label should be should be provided in the language of the page.

**Comment by davidgotlieb** [comment]
This has been fixed in production as it was a content issue. It may take up to an hour to be visible as the cache needs to clear.

**Comment by mmiron-HC** [comment]
@davidgotlieb Can you explain how all the things fixed in PROD will be maintained next release? Will OP remember everything they've fixed in Production and have to refix these if they haven't in Dev. I'm really curious how this is being managed.

**Comment by davidgotlieb** [comment]
@mmiron-HC The majority (if not all) of the issues that were flagged where it is happening on Dev but not Prod are related to content, no features. For example, the issues with the help section. Content would not be uploaded from Dev as it's our testing environment. If we needed to replace the content on Prod we'd restore from a backup.

**Comment by echouin** [comment]
@mmiron-HC recommend for closing. As noted previously, this is fixed on live prod website:
![image](https://user-images.githubusercontent.com/88099454/187461050-54024054-2ed7-4128-8f93-367bc7c0c458.png)

**Comment by sarasuri** [comment]
Verified (Sept 27, 2022) on both DEV and PROD, the issue is fixed.
Closing the issue.
